soc/nordic/nrf53/Kconfig

```diff
@@ -179,8 +181,6 @@ config SOC_NRF53_CPUNET_ENABLE
 	depends on !BT
 	default y if NRF_802154_SER_HOST
 	select SOC_NRF53_CPUNET_MGMT
-	select SOC_NRF_GPIO_FORWARDER_FOR_NRF5340 if \
-		$(dt_compat_enabled,$(DT_COMPAT_NORDIC_NRF_GPIO_FORWARDER))
```

Comment:
Same entry is here: https://github.com/zephyrproject-rtos/zephyr/blob/ec0837e3474df47312664070139724e678046176/boards/nordic/nrf7002dk/Kconfig#L26-L27 and it's also unneeded now.

Reply:
Fixed